database/mint.go

```diff
@@ -19,7 +18,7 @@ ON CONFLICT (one_row_id) DO UPDATE
         height = excluded.height 
 WHERE inflation.height <= excluded.height`
 
-	_, err := db.Sql.Exec(stmt, inflation.String(), height)
+	_, err := db.Sql.Exec(stmt, inflation, height)
```

Comment:
Would it be better to store `inflation` value as a string to avoid value overflow in the future?